### ISSUE TITLE
GD - 原理推导

### ISSUE THREAD

**Issue by junxnone** [junxnone/AI#106]
# Reference
- [梯度下降（Gradient Descent）小结](https://www.cnblogs.com/pinard/p/5970503.html)

# 梯度下降
Term | Description
-- | --
Learning rate | 梯度下降迭代的过程中，每一步沿梯度负方向前进的长度
Hypothesis function |  拟合输入样本
Loss function | 度量拟合的程度

### 代数法

假设函数 | 
损失函数 | 

### 矩阵法

**迭代公式**
![image](https://user-images.githubusercontent.com/2216970/69918537-2a042080-14ae-11ea-8676-662e0ea89a44.png)

**Comment by junxnone** [comment]
#89
